pkg/client/requestaccess_test.go

```diff
@@ -4,40 +4,49 @@ import (
 	"testing"
 
 	"github.com/storacha/go-libstoracha/capabilities/access"
+	"github.com/storacha/go-ucanto/core/invocation"
+	"github.com/storacha/go-ucanto/core/receipt/fx"
 	"github.com/storacha/go-ucanto/did"
 	ed25519 "github.com/storacha/go-ucanto/principal/ed25519/signer"
+	"github.com/storacha/go-ucanto/server"
+	uhelpers "github.com/storacha/go-ucanto/testing/helpers"
+	"github.com/storacha/go-ucanto/ucan"
 	"github.com/storacha/guppy/pkg/client"
+	"github.com/storacha/guppy/pkg/testing/helpers"
 	"github.com/stretchr/testify/require"
 )
 
 func TestRequestAccess(t *testing.T) {
 	t.Run("invokes `access/authorize`", func(t *testing.T) {
-		signer, err := ed25519.Generate()
-		require.NoError(t, err)
+		agentPrincipal := uhelpers.Must(ed25519.Generate())
 
-		account, err := did.Parse("did:mailto:example.com:alice")
-		require.NoError(t, err)
+		account := uhelpers.Must(did.Parse("did:mailto:example.com:alice"))
 
-		servicePrincipal, err := did.Parse("did:web:storage.example.com")
-		require.NoError(t, err)
+		invokedCapabilities := make([]ucan.Capability[access.AuthorizeCaveats], 0)
```

Comment:
hint: when initializing a slice with 0 length, you can also do
```
invokedCapabilities := []ucan.Capability[access.AuthorizeCaveats]{}
```
I find it a little less weird than the `0` argument to make, but that is of course very subjective.